src/components/reader/Page.tsx

```diff
@@ -120,17 +103,26 @@ const Page = React.forwardRef((props: IProps, ref: any) => {
         }
     }, [handleVerticalScroll]);
 
+    const imgStyle = imageStyle(settings);
+
     return (
-        <div ref={ref} style={{ margin: 'auto' }}>
+        <Box ref={ref} sx={{ margin: 'auto' }}>
             <SpinnerImage
                 src={`${src}?useCache=${useCache}`}
                 onImageLoad={onImageLoad}
                 alt={`Page #${index}`}
                 imgRef={imgRef}
-                spinnerClassName={`${classes.image} ${classes.loadingImage}`}
```

Comment:
Merging the two might have been wrong in the first place?

Reply:
Probably yes, cause they both might have conflicting css. Having different styling objects for both makes more sense